test/testenv/deployment.go

```diff
@@ -403,3 +404,33 @@ func (d *Deployment) DeployStandaloneWithLM(name string) (*enterprisev1.Standalo
 	}
 	return deployed.(*enterprisev1.Standalone), err
 }
+
+// DeployStandaloneWithIndexes deploys a standalone splunk enterprise instance on the specified testenv
+func (d *Deployment) DeployStandaloneWithIndexes(name string, indexesSecret string, volumeName, string, indexName string) (*enterprisev1.Standalone, error) {
+
+	volumeSpec := GenerateIndexVolumeSpec(volumeName, "https://s3-us-west-2.amazonaws.com", "splk-integration-test-bucket", indexesSecret)
+	indexSpec := GenerateIndexSpec(indexName, volumeName)
+
+	spec := enterprisev1.StandaloneSpec{
+		CommonSplunkSpec: enterprisev1.CommonSplunkSpec{
+			Spec: splcommon.Spec{
+				ImagePullPolicy: "IfNotPresent",
```

Comment:
Is this necessary to set this here?  Just wondering why the `ImagePullPolicy` matters and nothing else....

Reply:
I am sure it has no effect. It just something I followed from other places in code for consistency.